tests/test_game.py

```diff
@@ -23,3 +23,71 @@ def test_game_from_stock(self):
         self.game.deal_from_stock()
         self.assertEqual(len(self.game.waste), prev_waste_len + 1)
         self.assertEqual(len(self.game.stock), prev_stock_len - 1)
+
+    def test_move_tableau_pile(self):
+        # Setup: Ensure we have a known state
+        self.game.tableau[0] = [game.Card('K', 'spades', True)]
+        self.game.tableau[1] = [game.Card('Q', 'hearts', True)]
+        self.game.move_tableau_pile(0, 1)
```

Comment:
This testing is not passing, can you fix it? In fact, to move a K on top of a Q is an invalid move, it's the Q card that can be moved on top of a K.